models/card.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+const validator = require('validator');
+
+const cardSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    minlength: 2,
+    maxlength: 30,
+    required: true,
+  },
+  link: {
+    type: String,
+    required: true,
+    validate: {
+      validator(value) {
+        return validator.isUrl(value);
```

Comment:
Нужно исправить: при сохранении карточки происходит ошибка валидации.
```json
{
    "message": "Что-то пошло не так",
    "err": "card validation failed: link: validator.isUrl is not a function"
}
```
```suggestion
        return validator.isURL(value);
```